pillar.example

```diff
@@ -26,8 +26,9 @@ sshd_config:
   MaxSessions: 10
   RSAAuthentication: 'yes'
   PubkeyAuthentication: 'yes'
-  AuthorizedKeysCommand: '/usr/bin/sss_ssh_authorizedkeys'
-  AuthorizedKeysCommandUser: 'nobody'
+  AuthorizedKeysCommand: 
```

Comment:
If I'm not mistaken, you'll need to use snake case (asdf_asdf) here, because it's used in the Jinja template.